core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-analytic-app-misconfiguration/dot-experiments-analytic-app-misconfiguration.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { createComponentFactory, mockProvider, Spectator } from '@ngneat/spectator/jest';
+
+import { DotMessageService } from '@dotcms/data-access';
+import { DotEmptyContainerComponent } from '@dotcms/ui';
+
+import { DotExperimentsAnalyticAppMisconfigurationComponent } from './dot-experiments-analytic-app-misconfiguration.component';
+
+describe('DotExperimentsAnalyticAppMisconfigurationComponent', () => {
+    let spectator: Spectator<DotExperimentsAnalyticAppMisconfigurationComponent>;
+    const createComponent = createComponentFactory({
+        component: DotExperimentsAnalyticAppMisconfigurationComponent,
+        providers: [mockProvider(DotMessageService)]
+    });
+
+    beforeEach(() => {
+        spectator = createComponent();
+    });
+
+    it('should DotEmptyContainerComponent exist in the component', () => {
+        expect(spectator.query(DotEmptyContainerComponent)).toExist();
```

Comment:
Throw a comment about the integration then.

Reply:
Added 2 small test.

Reply:
I mean add a comment here to tell that this is being tested in other places.